ckan/plugins/core.py

```diff
@@ -71,6 +71,21 @@ def use_plugin(*plugins):
         unload(*plugins)
 
 
+class PluginImplementations(ExtensionPoint):
+
+    def __iter__(self):
+        '''
+        When we upgraded pyutilib on CKAN 2.9 the order in which
+        plugins were returned by `PluginImplementations` changed
+        so we use this wrapper to maintain the previous order
+        (which is the same as the ckan.plugins config option)
+        '''
+
+        iterator = super(PluginImplementations, self).__iter__()
+
+        return reversed(list(iterator))
```

Comment:
is there no `__reversed__` implementation we can use instead of creating a temporary list?

Reply:
You mean in the [parent class](https://github.com/PyUtilib/pyutilib/blob/5.7.3/pyutilib/component/core/core.py#L202)? Not AFAICT